libs/ledger-live-common/src/cal/partners.test.ts

```diff
@@ -88,7 +94,7 @@ describe("transformData", () => {
         },
       ];
 
-      const result = transformData(providersData, env as "prod" | "test");
+      const result = transformData(providersData);
```

Comment:
We need to tell the function what environment we want to use.

Reply:
Not anymore, the environment variable is called directly in the function.

Reply:
Changed.